cluster/examples/cluster-lock-000.json

```diff
@@ -0,0 +1,95 @@
+{
+  "cluster_definition": {
+    "name": "Obol Core Team 3 of 4",
+    "operators": [
+      {
+        "address": "",
+        "enr": "enr:-JG4QKxA4c5JCRVsxyHV2x1VYYKIjMOvOzrgvLn2MrxKKZExGK_iFy6gOjdt2ciMUuqc-CaCWCfsat2JpNzsV2pTELqGAYJE19-DgmlkgnY0gmlwhH8AAAGJc2VjcDI1NmsxoQJdmxjcoFYP8gfZiVVSZXbGhOLiu7bijKRac5hWEhcGg4N0Y3CCDhqDdWRwgg4u",
+        "nonce": 0,
+        "config_signature": null,
+        "enr_signature": null
+      },
+      {
+        "address": "",
+        "enr": "enr:-JG4QGcZTWSgiRXx0Wd_znS56PcjCUlY5Qk7Ja8KqlTeyvzBVcJ9Q-D7VyVX27smhtV-OkUwsn3rpgOk5qFbOYiQXKyGAYJFB9mzgmlkgnY0gmlwhH8AAAGJc2VjcDI1NmsxoQKGzfYZoH6ImLs20B7QHfOHt7pNB2FJX0P6eXMNIeF1yYN0Y3CCDhqDdWRwgg4u",
+        "nonce": 0,
+        "config_signature": null,
+        "enr_signature": null
+      },
+      {
+        "address": "",
+        "enr": "enr:-JG4QM2lHxfVhQX78Pby5--uuilN6jTFZMIhgxRjs1i51y6kPrx5JkvoiGCPN-XWJ4qzVKLrfxXvHGVhysO-7xPlHTSGAYJFBgEigmlkgnY0gmlwhH8AAAGJc2VjcDI1NmsxoQIMOdWK9zJOA9RfAtGSy_nirSERXfoxHxmdLRIWalQVJ4N0Y3CCDhqDdWRwgg4u",
+        "nonce": 0,
+        "config_signature": null,
+        "enr_signature": null
+      },
+      {
+        "address": "",
+        "enr": "enr:-JG4QG2QA-0OTnzUAmWtYuuAJ5QwhQRQTT0pTBuapBHgU75sBAnBZieYVR8fpHMzRc66sqgDYkhMJicLdz1ejhXjRV-GAYJFBnlQgmlkgnY0gmlwhH8AAAGJc2VjcDI1NmsxoQORVX5d5U7HULLj_qfh0qP6RrO35tieBwJcKGj4GNBaiYN0Y3CCDhqDdWRwgg4u",
+        "nonce": 0,
+        "config_signature": null,
```

Comment:
is it possible to populate these "null" fields?

Reply:
These are examples of real world locks. I didn't generate them.

Reply:
Where did you get them from?

Reply:
our cluster, charon create cluster, and a user's cluster